satpy/dataset/dataid.py

```diff
@@ -72,6 +72,246 @@ def __repr__(self):
         return '<' + str(self) + '>'
 
 
+class FrequencyDoubleSideBandBase(NamedTuple):
```

Comment:
This class and the next three are really specific to MHS/AMSU, should we put them in the mhs reader for now instead?

Reply:
Hmmm, you have a point. But I was hoping to add MWS support soon, and perhaps also ATMS. And AWS (same file format as MWS).